go.mod

```diff
@@ -3,32 +3,23 @@ module github.com/cilium/hubble
 go 1.13
 
 require (
-	github.com/cilium/cilium v1.7.0-rc2.0.20200225062438-9e30278230ce
-	github.com/fatih/color v1.7.1-0.20181010231311-3f9d52f7176a // indirect
+	github.com/cilium/cilium v1.7.0-rc2.0.20200225155839-f0d65d667e33
 	github.com/francoispqt/gojay v1.2.13
 	github.com/go-openapi/strfmt v0.19.3
 	github.com/gogo/protobuf v1.3.0
 	github.com/golang/protobuf v1.3.2
 	github.com/google/gopacket v1.1.17
 	github.com/google/gops v0.3.6
-	github.com/hashicorp/golang-lru v0.5.3
-	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
-	github.com/miekg/dns v1.1.22 // indirect
+	github.com/hashicorp/golang-lru v0.5.1
 	github.com/mitchellh/protoc-gen-go-json v0.0.0-20200113165135-fd297ce346f1
-	github.com/pelletier/go-toml v1.4.0 // indirect
 	github.com/prometheus/client_golang v1.2.0
 	github.com/sirupsen/logrus v1.4.2
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.6.1
 	github.com/stretchr/testify v1.4.0
-	google.golang.org/genproto v0.0.0-20190801165951-fa694d86fc64 // indirect
 	google.golang.org/grpc v1.23.1
 	k8s.io/apimachinery v0.17.3
 )
 
-replace (
-	github.com/miekg/dns => github.com/cilium/dns v1.1.4-0.20190417235132-8e25ec9a0ff3
```

Comment:
I think this could bite us at some point. This is a dependency of cilium and `go get` basically does not read replace directives outside the main module. In effect, this means that this dependency for cilium is different that the one that cilium actually expects. See the diff in `vendor/modules.txt`:
```
- # github.com/miekg/dns v1.1.22 => github.com/cilium/dns v1.1.4-0.20190417235132-8e25ec9a0ff3
+ # github.com/miekg/dns v1.0.14
```
I think that the only real fix for this, if these forks are "permanent", is to directly refer to the forks within cilium's codebase rather than using a replace directive (which is a feature intended for dev purposes; not the way it is used in cilium now (see [FAQ](https://github.com/golang/go/wiki/Modules#when-should-i-use-the-replace-directive)).